### PR TITLE
Update to new plugins.

### PR DESCRIPTION
**Pull request by Trao-X**
Update the README file in order to mention the new plugins.

### CODE REVIEW

**Review by Tolga1452 — CHANGES_REQUESTED** [review]
You have an idea about ordering? Why they are at top bro 💀

**Comment by Trao-X** [comment]
what

**Comment by Trao-X** [comment]
wdym bro

**Comment by Tolga1452** [comment]
> wdym bro

Maybe you should add new ones to end of the list instead of at top?

**Comment by Trao-X** [comment]
done 🫠

File: README.md [hunk 1]
```diff
@@ -91,3 +91,5 @@ When you want to update your Discord client to the latest version of this fork,
 - **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into a catgirl message.
 - **[Talk In Reverse](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/talkInReverse):** Reverses the message content before sending it.
 - **[God Mode](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/godMode):** Get all permissions (client-side).
+- **[Copy Emoji as Formatted String](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CopyEmojiAsFormattedString)** Adds a button to copy an emoji as a formatted string!
+- **[Custom App Icons](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CustomAppIcons)** Gives the ability to upload custom (In-)App Icons.
```
Comment (Tolga1452) [inline README.md:95]: Use the original plugin names 😭
```suggestion
- **[CopyEmojiAsFormattedString](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CopyEmojiAsFormattedString)** Adds a button to copy an emoji as a formatted string!
- **[CustomAppIcons](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CustomAppIcons)** Gives the ability to upload custom (In-)App Icons.
```

**Review by Tolga1452 — CHANGES_REQUESTED** [review]
God

File: README.md (hunk 1 above)
Comment (Trao-X) [reply README.md:95]: It's confusing that the other plugins have spaces, but these not. I made a pull request last night to make the other plugins without spaces, but it was denied.

Comment (Tolga1452) [reply README.md:95]: > It's confusing that the other plugins have spaces, but these not. I made a pull request last night to make the other plugins without spaces, but it was denied.

Plugin names are personal choices and not confusing at all. Just copy the original plugin name in this case

**Review by Tolga1452 — APPROVED** [review]

**Pull request merged**